src/content/containers/URLManager.js

```diff
@@ -42,7 +42,7 @@ class URLManager extends PureComponent {
     const { urlSetupDone } = this.props;
 
     this._updateState();
-    window.onpopstate = () => this._updateState();
+    window.addEventListener('popstate', () => this._updateState(), false);
```

Comment:
This is not especially worrying for this component, but for correctness we should remove the listener in
`componentWillUnmount`.

Also the last `false` is unnecessary.